### ISSUE TITLE
CSV Importer: cURL error 6: Could not resolve host with docker-compose

### ISSUE THREAD

**Issue by AlexKnowsIt** [firefly-iii/firefly-iii#4210]
I am running the current docker image (latest tag)

**Description**
First off: Thanks for the amazing software! I stumbled yesterday about firefly and the setup was straight forwar over docker-compose. I tried today to enable the CSV Importer and I am stuck already for hours without knowing what I am doing wrong.
As I said I used docker-compose to set-up firefly and tried to use the docker image of the CSV Importer to import data. The Webserver is also opening but I get the error message: `cURL error 6: Could not resolve host: %20http (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for %20http://fireflyiii:8080/api/v1/about`
I changed the URI already in every way that seems logic to me so I used the internal docker URL that I received from this command `docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' CONTAINER`
I tried to not start the container stand-alone but with docker-compose so that the services are in the same virtual network. I used the http://fireflyiii:8080 URI but so far nothing worked. I also tried running it inline but I get the same error message in the CL.

**My guess**
I think this has to do something with the virtual network that got created trough docker-compose. But I am also not sure if the API is working. It seems I can't browse the endpoints with the browser and sadly I have not the possibility to install postman on my machine.

Here is my latest docker-compose file (with my trial to insert the csvimporter into the same virtual network):

version: '3.3'

services:
  fireflyiii:
    image: jc5x/firefly-iii:latest
    volumes:
      - firefly_iii_upload:/var/www/html/storage/upload
    env_file: .env
    ports:
      - 8030:8080
    depends_on:
      - fireflyiiidb
  fireflyiiidb:
    image: yobasystems/alpine-mariadb:latest
    environment:
      - MYSQL_RANDOM_ROOT_PASSWORD=yes
      - MYSQL_USER=firefly
      - MYSQL_PASSWORD=secret_firefly_password
      - MYSQL_DATABASE=firefly
    volumes:
      - firefly_iii_db:/var/lib/mysql
  csvimporter:
    image: fireflyiii/csv-importer:latest
    environment: 
      - FIREFLY_III_ACCESS_TOKEN=eyJ...
      - FIREFLY_III_URL=http://172.18.0.4:8080
    ports:
      - 8031:8080

volumes:
   firefly_iii_upload:
   firefly_iii_db:

The .env file is the same as from the repo, I just changed the language settings.

**Comment by JC5** [comment]
It looks like a space somewhere in the URL. Does it always say "%20" at the start of the URL? Do you have a longer stacktrace?

**Comment by AlexKnowsIt** [comment]
So I was working the last hours with trial and error to see where the weird sign at the beginning was coming from and I deccided to press the reauthenticate button from the webserver multiple times and out of nowhere it worked. I did not change anything in my setup so maybe this needs to be done because the services start not exactly at the same time.
Thank you for your fast help and this amazing piece of software!
